client/my-sites/domains/domain-management/edit/transfer.jsx

```diff
@@ -43,26 +44,76 @@ class Transfer extends React.PureComponent {
 		this.props.paymentSettingsClick( this.props.domain );
 	};
 
+	restartTransfer = () => {
+		const { domain, selectedSite, translate } = this.props;
+
+		restartInboundTransfer( selectedSite.ID, domain.name, ( error, result ) => {
+			if ( result ) {
+				this.props.successNotice( translate( 'The transfer has been successfully restarted.' ), {
+					duration: 5000,
+				} );
+				fetchDomains( selectedSite.ID );
+			} else {
+				this.props.errorNotice(
+					error.message || translate( 'We were unable to restart the transfer.' ),
+					{
+						duration: 5000,
+					}
+				);
+			}
+		} );
+	};
+
+	getCancelledContent() {
+		const { domain, translate } = this.props;
+
+		return (
+			<Card>
+				<div>
+					<p className="edit__transfer_text_fail">{ translate( 'Domain Transfer Failed' ) }</p>
+					<p>
+						{ translate(
+							'We were unable to verify the transfer of {{strong}}%(domain)s{{/strong}}. The domain ' +
+								'authorization code from you current registrar was not provided to initiate the transfer. ' +
+								'{{a}}Learn More{{/a}}.',
+							{
+								components: {
+									strong: <strong />,
+									a: <a href={ '#' } />,
+								},
+								args: { domain: domain.name },
+							}
+						) }
+					</p>
+				</div>
+				<div>
+					<Button onClick={ this.restartTransfer }>
+						{ this.props.translate( 'Start Transfer Again' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **11** times:
`translate( 'Start my Guided Transfer' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).